### PR TITLE
Bump pisces

### PR DESCRIPTION
**Pull request by TMiguelT**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by TMiguelT** [comment]
Notably the linter is always going to fail here because we can't add tests since `dotnet` isn't available as a conda package.

**Comment by TMiguelT** [comment]
...unless I skip the tests entirely

**Comment by npavlovikj** [comment]
Btw, the version of `VariantQualityRecalibration` in `build.sh` is 5.2.7.47. Shouldn't this be updated to `5.2.10.49` as well?

**Comment by npavlovikj** [comment]
Also, as you can see from the linting information, for GPL licensed software, the license file needs to be packaged as well. If the license is not in the tarball, then please copy it to the recipe directory and add it accordingly.

File: recipes/pisces/meta.yaml [hunk 1]
```diff
@@ -29,3 +30,9 @@ about:
   home: "https://github.com/Illumina/Pisces"
   license: GPLv3
   summary: "Somatic and germline variant caller for amplicon data. Recommended caller for tumor-only workflows."
+
+
+extra:
+  skip-lints:
+    # dotnet isn't in Conda, so we can't test pisces
+    - missing_tests
```
Comment (dpryan79) [inline recipes/pisces/meta.yaml]: Can you try making mono a dependency and then using it to run tests?

Comment (TMiguelT) [reply recipes/pisces/meta.yaml]: Pisces 5.2 targets the dotnet framework only, not mono: https://github.com/Illumina/Pisces#system-requirements

**Comment by npavlovikj** [comment]
Since the License file is in the recipe directory now, please use:
`license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'`
instead.

File: recipes/pisces/meta.yaml (hunk 1 above)
Comment (dpryan79) [reply recipes/pisces/meta.yaml]: Mono is an open source implementation of the dot net framework.

**Comment by bgruening** [comment]
@npavlovikj  I think if its in the recipe dir we do not need to prefix it. LICENSE.txt should just work.

**Comment by TMiguelT** [comment]
It is failing to find the file though. Any ideas why?

**Comment by npavlovikj** [comment]
@TMiguelT , the error message you are getting is:
` 05:50:06 BIOCONDA INFO (OUT) ValueError: License file given in about/license_file (/opt/recipe/README.txt) does not exist in source root dir or in recipe root dir (with meta.yaml)`
The license file is named "LICENSE.txt" and not "README.txt", so please change that.

File: recipes/pisces/meta.yaml (hunk 1 above)
Comment (TMiguelT) [reply recipes/pisces/meta.yaml]: Yes but pisces targets dotnet core, not the dotnet framework, so mono won't help.

**Comment by npavlovikj** [comment]
> @npavlovikj I think if its in the recipe dir we do not need to prefix it. LICENSE.txt should just work.

Thanks @bgruening - I use "environ" due to habit, but you are correct that "LICENSE.txt" is sufficient too.

**Comment by dpryan79** [comment]
Thanks @TMiguelT!

**Pull request merged**